.circleci/config.yml

```diff
@@ -283,6 +283,7 @@ jobs:
           tar -zxvf golang.tar.gz -C /tmp
           sudo rm -rf ${GOROOT}
           sudo mv /tmp/go ${GOROOT}
+      - run: GO111MODULE=on $GO mod vendor
```

Comment:
what's the reasoning behind removing the vendor folder just to revendor everything as part of the CI/build process?